packages/client/src/project/entities.ts

```diff
@@ -13,6 +13,15 @@ export function getProjectEntities (projectId: string, apiClient: AxiosInstance)
       const entities = res.data;
       return wrapEntityArray(entities);
     },
+    async patch(entity: Entity): Promise<WrappedEntity> {
+      const entityId = entity._id;
+      delete entity._id;
```

Comment:
Avoid mutating arguments supplied by the user, this can lead to unexpected bugs (e.g. the developer may not be aware the `_id` field was removed from their entity object and may try to reuse only to find it gone.
If you want to change properties, create a copy of the object:
```js
const _entity = { ...entity };
const entityId = _entity._id;
delete _entity._id; // mutated copy
```

Reply:
Done